modules/nf-core/sentieon/qualcal/tests/nextflow.config

```diff
@@ -0,0 +1,9 @@
+env {
+    // NOTE This is how nf-core/sarek users will use Sentieon in real world use
```

Comment:
Not just `nf-core/sarek` users, could be other places too.

Reply:
would like to update this in on e sweep across all modules, but can do it right now too

Reply:
Fine, feel free to do it in one go.